config/config.go

```diff
@@ -222,9 +222,11 @@ func (c *Config) Merge(o *Config) *Config {
 		r.Wait = r.Wait.Merge(o.Wait)
 	}
 
-	r.Once = o.Once
+	if o.BlockQueryWaitTime != nil {
+		r.BlockQueryWaitTime = o.BlockQueryWaitTime
+	}
```

Comment:
This check would probably be useful above in `Copy()` as well https://github.com/hashicorp/consul-template/blob/8ab9628fb643c435a77d1c8583674e0aff8fbc79/config/config.go#L152